src/test/resources/data/100Eori.json

```diff
@@ -0,0 +1,6 @@
+{
+  "date":"2024-02-08",
+  "authType": "UKNW",
+  "eoris" : ["GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687","GB974163550126687"
```

Comment:
Please can you run these through a formatter.

Reply:
Formatted the json.